python/tvm/relay/frontend/caffe.py

```diff
@@ -637,6 +638,22 @@ def convert_embed(self, op):
 
         return out
 
+    def convert_power(self, op):
+        """Convert Power layer"""
+        inputs = op.bottom
+        in_expr = self.exp_tab.get_expr(inputs[0])
+        power = _expr.const(op.power_param.power)
+        scale = _expr.const(op.power_param.scale)
+        shift = _expr.const(op.power_param.shift)
+
+        if scale != 1.0:
+            out = _op.multiply(in_expr, scale)
+        if shift != 0.0:
+            out = _op.add(out, shift)
```

Comment:
if `scale` is 1.0, line 650 won't be executed.
Then `out` is undefined which causes an undefined reference error.